Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -1378,8 +1379,8 @@ void ScheduleReads(u64 offset, u32 length, const DiscIO::Partition& partition, u
             "Schedule reads: offset=0x%" PRIx64 " length=0x%" PRIx32 " address=0x%" PRIx32, offset,
             length, output_address);
 
-  // The DVD drive's minimum turnaround time on a command, based on a hardware test.
```

Comment:
This line seems to indicate there was a hardware test before.  Any idea what changed?

Reply:
If I'm understanding it correctly, the earlier hardware test determined that 300 µs was the fastest that *any* command can complete. This is a more complex command, so it makes sense that its latency would be higher than the latency of the simplest command.

Reply:
Hmm, I guess we can only ever approximate this.  What effect does this have on games that don't need this change? Longer load times?

Reply:
Yes, but just a tiny bit.